loggroup-lambda-connector/sam/template.yaml

```diff
@@ -41,7 +41,7 @@ Parameters:
 
   DestinationArnValue:
     Type: String
-    Default: "arn:aws:lambda:us-east-1:123456789000:function:TestLambda"
+    Default: "arn:aws:lambda:us-west-2:123456789000:function:TestLambda"
```

Comment:
This is not the KBS AWS account is it? I would expect the arn to start with arn:aws:lambda:us-west-2:529513974030